hmda/src/main/scala/hmda/validation/rules/lar/validity/2025/V619_2.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity._2025
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.ValidationResult
+import hmda.validation.rules.EditCheck
+
+object V619_2 extends EditCheck[LoanApplicationRegister] {
```

Comment:
The last test file we have for V619_2  is from 2022, do we not need tests for each year for V619_2?

Reply:
We don't need to , but it'd probably be the responsible thing 👀